pkgs/tools/admin/pulumi/default.nix

```diff
@@ -17,9 +17,10 @@ in stdenv.mkDerivation {
   installPhase = ''
     mkdir -p $out/bin
     cp * $out/bin/
+    wrapProgram $out/bin/pulumi --set LD_LIBRARY_PATH "${stdenv.cc.cc.lib}/lib"
```

Comment:
Since we already have `autoPatchelfHook`, it probably would have been enough to just include `stdenv.cc.cc` in the `buildInputs`.

Reply:
I tried that but it didn't work. Maybe I was doing something wrong but when I ran "ldd" against the binary it didn't show libstdc++. The error happened at runtime while running it against a python project. It looked like it was trying to load the library at that stage. Not sure though, no expert on Nix nor Pulumi 😅

Reply:
It might do `dlopen`.

Reply:
@Mic92 I imagine, working with libstdc++ using `dlopen` would be quite difficult because of the mangling.

Reply:
It could `dlopen` a c++ shared library that in turn require `libstdc++`.
Anyway using `LD_DEBUG=libs` as environment variable would show where it is loaded.